torchaudio/functional.py

```diff
@@ -458,14 +458,17 @@ def phase_vocoder(
     factor of ``rate``.
 
     Args:
-        complex_specgrams (Tensor): Dimension of `(..., freq, time, complex=2)`
+        complex_specgrams (Tensor): Either a real tensor of dimension of `(..., freq, time, complex=2)`
+            or a tensor of dimension `(..., freq, time)` with complex dtype.
         rate (float): Speed-up factor
         phase_advance (Tensor): Expected phase advance in each bin. Dimension of (freq, 1)
 
     Returns:
-        Tensor: Complex Specgrams Stretch with dimension of `(..., freq, ceil(time/rate), complex=2)`
+        Tensor: Complex Specgrams Stretch with either a real dtype and dimension of
+                `(..., freq, ceil(time/rate), complex=2)` or
+                a complex dtype and dimension of `(..., freq, ceil(time/rate))`.
 
-    Example
+    Example - old API
```

Comment:
From the discussion above, it is not clear which API is "old" and which one is "new". More direct to say "with real tensor input" and "with complex tensor input"

Reply:
yeah I agree that's more clear! updated